agent/bin/test_install

```diff
@@ -49,7 +52,12 @@ function install_petclinic (
 # hash, though, because Travis detects that as a secret. :( 8 digits should be
 # enough to make a collision extremely unlikely.)
 install_petclinic "spring-projects/spring-petclinic" 9cb8dde9
+../gradlew :pomupdater:run -PmainClass=com.appland.appmap.util.PomUpdater \
+  -PpomFile="$PWD/build/fixtures/spring-petclinic/pom.xml" \
+  -PannotationJar="${ANNOTATION_JAR}"
```

Comment:
I'm not sure I understand why we need to add this on petclinic. Also, is this something our users will need to be to do, too? Will they need to use pomupdater for that?

Reply:
As part of setup, we add a test file that uses the `@NoAppMap` annotation. To compile the tests, we need the annotation jar as a dependency.

A user will certainly need to add the dependency somehow, but I don't think they'll look to us for help doing that. I guess they can use `PomUpdater` if they find it, but I think we should just leave it as an internal tool for now.

Reply:
Ok, thanks.